src/components/editor/SinopiaResourceTemplates.jsx

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 Stanford University see Apache2.txt for license
+
+import React, {Component} from 'react'
+import BootstrapTable from 'react-bootstrap-table-next'
+const SinopiaServer = require('sinopia_server')
+const instance = new SinopiaServer.LDPApi()
+instance.apiClient.basePath = 'http://localhost:8080'
+
+class SinopiaResourceTemplates extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isLoading: false,
+      groupData: [],
+      templatesForGroup: []
+    }
+  }
+
+  async componentDidMount() {
+    const groupPromise = new Promise((resolve) => {
+      resolve(instance.getBaseWithHttpInfo())
+    }).then((data) => data).catch(() => {})
```

Comment:
(trying to understand what I always screw up:  

- so this new promise is necessary b/c we need to silence the error from instance.getBaseWithHttpInfo() with the empty catch block, but still pass to fulfillGroupPromise() regardless?

Reply:
Yes to the former. we need to silence the error from instance.getBaseWithHttpInfo() with the empty catch block, but still pass to fulfillGroupPromise() regardless. Otherwise the integration test fails.